qemu/tests/cfg/rh_kernel_update.cfg

```diff
@@ -1,12 +1,12 @@
-- rh_kernel_update:
+ rh_kernel_update:
```

Comment:
delete `-` by mistake?

Reply:
@yanan-fu `backup_image` rename can be avoided, rather as suggested we can keep the same in avocado-vt patch.